project/scalastyle-config.xml

```diff
@@ -1,14 +1,14 @@
 <scalastyle>
```

Comment:
Just reformatting that I did after reverting some quality-of-life config changes.